tidb-cloud/tidb-cloud-tls-connect-to-dedicated-tier.md

```diff
@@ -0,0 +1,358 @@
+---
+title: TLS connection to Dedicated Tier
+summary: Introduce TLS connection in TiDB Dedicated Tier.
+---
+
+# TLS connection to Dedicated Tier
+
+On TiDB Cloud, TLS connections are one of the basic security practices for connecting to Dedicated Tier clusters. We support you to configure multiple TLS connections from client, application and development tools to protect your data transmission security, mainly including: supporting MySQL CLI , MyCLI , JDBC, Python, Go and Node.js.
```

Comment:
I don't like the last sentence. TiDB is MySQL compatible so any MySQL-compatible client is able to connect to TiDB. Don't give users a feeling that we only support these clients. We just give them some examples here.